pages/main.js

```diff
@@ -9,6 +10,66 @@ const Modal = require('../elements/modal')
 
 const modal = Modal()
 
+css('dat-colors')
+
+var prefix = css`
```

Comment:
also, should be a `const`, see https://github.com/datproject/dat-desktop/pull/108#discussion_r91483407